web/src/app/components/loi-editor/loi-editor.component.ts

```diff
@@ -56,9 +61,22 @@ export class LoiEditorComponent {
   }
 
   clearLois() {
-    for (const loi of this.lois) {
-      this.dataStoreService.deleteLocationOfInterest(this.survey.id, loi.id);
-    }
+    const dialogRef = this.dialog.open(JobDialogComponent, {
+      data: {
+        dialogType: DialogType.DeleteLois,
+        surveyId: this.survey.id,
+        jobId: this.job.id,
+      },
+      panelClass: 'small-width-dialog',
+    });
+
+    dialogRef.afterClosed().subscribe(async (result: DialogData) => {
+      if (!result) return;
+
+      for (const loi of this.lois) {
```

Comment:
This logic should really be in the service, not in the component.. Let's add a TODO for now?